CMakeLists.txt

```diff
@@ -448,8 +448,9 @@ if(ENABLE_MASON)
   set(TBB_LIBRARIES ${MASON_PACKAGE_tbb_LDFLAGS})
 
   mason_use(libshp2 VERSION ${MASON_LIBSHP_VERSION})
-  set(LIBSHAPEFILE_INCLUDE_DIR ${MASON_PACKAGE_libshp2_INCLUDE_DIRS})
-  set(LIBSHAPEFILE_LIBRARY ${MASON_PACKAGE_libshp2_LDFLAGS})
+  target_include_directories(UTIL PRIVATE ${MASON_PACKAGE_libshp2_INCLUDE_DIRS})
+  set(MAYBE_SHAPEFILE ${MASON_PACKAGE_libshp2_LDFLAGS})
```

Comment:
I don't understand this - I thought `MAYBE_SHAPEFILE` contains the (optional) shapefile lib, e.g. as in `libshapefile.so`? Why do we add linker flags here, too?